addons/sale_timesheet/views/project_update_templates.xml

```diff
@@ -4,35 +4,48 @@
         <!--As this template is rendered in an html field, the spaces may be interpreted as nbsp while editing. -->
         <xpath expr="//div[@name='milestone']" position="before">
 <br/>
-<div t-if="project.allow_billable and 'data' in services and len(services['data']) > 0">
-<h3><b><u>Sold</u></b></h3>
+<div t-if="show_sold">
+<h3 style="font-weight: bolder"><u>Sold</u></h3>
 <table class="table table-bordered">
 <tbody>
 <thead>
-<td>Service</td>
-<td>Sold</td>
-<td>Effective</td>
-<td>Remaining</td>
+<td style="font-weight: bolder">Service</td>
+<td style="font-weight: bolder">Sold</td>
+<td style="font-weight: bolder">Effective</td>
+<td style="font-weight: bolder">Remaining</td>
 </thead>
 <tr t-foreach="services['data']" t-as="service">
-<td><t t-esc="service['name']"/></td>
-<td><t t-esc="service['sold_value']"/> <t t-esc="service['uom']"/></td>
-<td><t t-esc="service['effective_value']"/> <t t-esc="service['uom']"/></td>
-<td><t t-esc="service['remaining_value']"/> <t t-esc="service['uom']"/></td>
+<td>
+<span t-if="service['is_unit']"><i><t t-esc="service['name']"/></i></span>
+<span t-else=""><t t-esc="service['name']"/></span>
+</td>
+<td style="text-align: right; vertical-align: middle;">
+<span t-if="service['is_unit']"><i><t t-esc="format_lang(service['sold_value'], 1)"/> <t t-esc="service['unit']"/></i></span>
+<span t-else=""><t t-esc="format_lang(service['sold_value'], 1)"/> <t t-esc="service['unit']"/></span>
+</td>
+<td style="text-align: right; vertical-align: middle;">
+<span t-if="service['is_unit']"><i><t t-esc="format_lang(service['effective_value'], 1)"/> <t t-esc="service['unit']"/></i></span>
+<span t-else=""><t t-esc="format_lang(service['effective_value'], 1)"/> <t t-esc="service['unit']"/></span>
+</td>
+<td style="text-align: right; vertical-align: middle;">
+<span t-if="service['is_unit']"><i><t t-esc="format_lang(service['remaining_value'], 1)"/> <t t-esc="service['unit']"/></i></span>
+<span t-else=""><t t-esc="format_lang(service['remaining_value'], 1)"/> <t t-esc="service['unit']"/></span>
+</td>
```

Comment:
Couldn't we rather use this code architecture:
```xml
<t t-if="service['is_unit']">
<td><span><i><t t-esc="service['name']"/></i></span></td>
<td><span><i><t t-esc="format_lang(service['sold_value'], 1)"/> <t t-esc="service['unit']"/></i></span></td>
...
</t>
<t t-else="">
<td><span><t t-esc="service['name']"/></span></td>
<td><span><t t-esc="format_lang(service['sold_value'], 1)"/> <t t-esc="service['unit']"/></span></td>
...
</t>
```

This would at least reduce the number of tests on service['is_unit']